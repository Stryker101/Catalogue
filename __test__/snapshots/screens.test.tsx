import React from "react";
import renderer from 'react-test-renderer';
import LoginScreen from "../../src/screens/login.screen";
import DashboardScreen from "../../src/screens/dashboard.screen";
import AddInventoryScreen from "../../src/screens/addinventory.screen";
import EditInventoryScreen from "../../src/screens/editinventory.screen";
import { NativeBaseProvider } from "native-base";


test('login screen renders correctly', () => {
    const tree = renderer
      .create(<NativeBaseProvider><LoginScreen/></NativeBaseProvider>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('dashboard screen renders correctly', () => {
    const tree = renderer
      .create(<NativeBaseProvider><DashboardScreen/></NativeBaseProvider>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('add inventory renders correctly', () => {
    const tree = renderer
      .create(<NativeBaseProvider><AddInventoryScreen/></NativeBaseProvider>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('edit inventory renders correctly', () => {
    const tree = renderer
      .create(<NativeBaseProvider><EditInventoryScreen/></NativeBaseProvider>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });