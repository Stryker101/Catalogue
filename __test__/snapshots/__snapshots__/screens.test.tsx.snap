// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`add inventory renders correctly 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    [
      {
        "flex": 1,
      },
      undefined,
    ]
  }
/>
`;

exports[`dashboard screen renders correctly 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    [
      {
        "flex": 1,
      },
      undefined,
    ]
  }
/>
`;

exports[`edit inventory renders correctly 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    [
      {
        "flex": 1,
      },
      undefined,
    ]
  }
/>
`;

exports[`login screen renders correctly 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    [
      {
        "flex": 1,
      },
      undefined,
    ]
  }
/>
`;

exports[`renders correctly 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    [
      {
        "flex": 1,
      },
      undefined,
    ]
  }
/>
`;

exports[`renders correctly 2`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    [
      {
        "flex": 1,
      },
      undefined,
    ]
  }
/>
`;
